begin/cmdline.py

```diff
@@ -58,10 +58,16 @@ def from_name(self, name, default=NODEFAULT, section=None):
         "Get default value from argument name"
         if len(self._parser.sections()) > 0:
             section = self._section if section is None else section
-            try:
-                default = self._parser.get(section, name)
-            except (configparser.NoSectionError, configparser.NoOptionError):
-                pass
+            if isinstance(section, list):
```

Comment:
I'd test if `not isinstance(section, str)`, as users can pass a tuple or a set here, too.